# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gimp/end-decls.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

http://svn.gnome.org/viewcvs/gimp?view=revision&revision=23376

diff -Nur gimp-2.4.0~rc1/libgimpwidgets/gimpcolorprofilestore.h gimp-2.4.0~rc1.new/libgimpwidgets/gimpcolorprofilestore.h
--- gimp-2.4.0~rc1/libgimpwidgets/gimpcolorprofilestore.h	2007-08-15 17:57:04.000000000 -0400
+++ gimp-2.4.0~rc1.new/libgimpwidgets/gimpcolorprofilestore.h	2007-08-25 18:00:39.000000000 -0400
@@ -64,5 +64,6 @@
                                                   const gchar           *filename,
                                                   const gchar           *label);
 
+G_END_DECLS
 
 #endif  /* __GIMP_COLOR_PROFILE_STORE_H__ */
